pyproject.toml

```diff
@@ -13,6 +13,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.10"
+litellm = "^1.24.6"
```

Comment:
If litellm is to be used in the `cli` code only, please add it to the corresponding CLI group instead of the generic bucket of dependencies. Please do not edit this section directly with your text editor, but do the following instead (from the project's root folder):
```bash
poetry remove litellm
poetry add --group=cli litellm
```

Reply:
The plan is to use LiteLLM also for the server implementation. Wouldn't it be smarter to just leave LiteLLM than as global dependency?